docs/source/guide/guide-zne.rst

```diff
@@ -637,7 +637,7 @@ Typically, the ``self.__init__`` method must be overridden.
 
 A new non-adaptive method can instead be derived from the abstract :class:`.BatchedFactory` class.
 In this case it is usually sufficient to override only the ``self.__init__`` and
-the ``self.reduce`` methods, which are responsible for the initialization and for the
+the ``self.extrapolate`` methods, which are responsible for the initialization and for the
```

Comment:
5 lines above there is also this line:
```
``self.next``, ``self.push``, ``self.is_converged``, ``self.reduce``, etc. , 
```
which should become:
```
``self.next``, ``self.push``, ``self.is_converged``, ``self.extrapolate``, etc. , 
```

Reply:
The adaptive factory was not modified, so should it not stay `self.reduce`?

Reply:
Good point. Your are right!